projects/train/train/callbacks.py

```diff
@@ -18,13 +18,11 @@ def on_train_end(self, trainer, pl_module):
             self.best_model_path, arch=pl_module.model, metric=pl_module.metric
         )
 
-        datamodule = trainer.datamodule
-        kernel_size = int(
-            datamodule.hparams.kernel_length * datamodule.sample_rate
-        )
-        sample_input = torch.randn(1, datamodule.num_ifos, kernel_size)
-        model = module.model.to("cpu")
-        trace = torch.jit.trace(model, sample_input)
+        device = pl_module.device
```

Comment:
Glad this worked. Just wan't to confirm that this doesn't break the BBH case. Could you double check this? (we should probably find a way to incorporate this into tests!)

Reply:
Confirmed that both the BNS and BBH case work